{"name": "Guest explores Ports of Departure", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\guest_explore_ports_steps.py\", line 29, in verify_port_in_middle\n    context.app.port_of_call_page.verify_port_in_middle()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\port_of_call.py\", line 22, in verify_port_in_middle\n    self.wait_for_element_appear(*self.PIN)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\base_page.py\", line 42, in wait_for_element_appear\n    self.wait.until(EC.presence_of_element_located(locator))\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given a Guest", "status": "passed", "start": 1597964030090, "stop": 1597964031558}, {"name": "And I am on Homepage", "status": "passed", "start": 1597964031558, "stop": 1597964033172}, {"name": "And I navigated to \"Ports\" page", "status": "passed", "start": 1597964033173, "stop": 1597964037279}, {"name": "When I search for \"Honolulu\" port", "status": "passed", "start": 1597964037279, "stop": 1597964042968}, {"name": "Then Map zoomed to show selected port", "status": "passed", "start": 1597964042969, "stop": 1597964043664}, {"name": "And Port is on the middle of the map", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\guest_explore_ports_steps.py\", line 29, in verify_port_in_middle\n    context.app.port_of_call_page.verify_port_in_middle()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\port_of_call.py\", line 22, in verify_port_in_middle\n    self.wait_for_element_appear(*self.PIN)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\ncl_qa_challenge_dt_12_12_2019\\pages\\base_page.py\", line 42, in wait_for_element_appear\n    self.wait.until(EC.presence_of_element_located(locator))\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1597964043665, "stop": 1597964062077}, {"name": "And Port displayed as \"Port Of Departure\"", "status": "skipped", "start": 1597964064877, "stop": 1597964064877}], "start": 1597964024930, "stop": 1597964064877, "uuid": "023b3607-6336-4bc4-b5e0-e5bbd111dc98", "historyId": "ce3be5c05a6ec998e99599a81305feef", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Guest explore Ports of Departure"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}